client/components/social-buttons/google.js

```diff
@@ -34,39 +34,37 @@ export default class GoogleLoginButton extends Component {
 		}
 
 		return new Promise( resolve => {
-			loadScript( 'https://apis.google.com/js/platform.js', () => resolve( window.gapi ) );
+			loadScript( 'https://apis.google.com/js/api.js', () => resolve( window.gapi ) );
```

Comment:
Both allow us to authenticate the user with their Google account, but `platform.js` is larger:

```bash
~ ☯ curl -s https://apis.google.com/js/platform.js | wc -m
   40156
~ ☯ curl -s https://apis.google.com/js/api.js | wc -m
   12313
```